dynamic-demo-plugin/package.json

```diff
@@ -9,13 +9,14 @@
     "build-plugin-sdk": "yarn --cwd ../frontend build-plugin-sdk && rm -rf node_modules/@openshift-console && yarn install --check-files",
     "http-server": "./http-server.sh dist",
     "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c i18next-parser.config.js",
-    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}' -I '/node_modules/(?!(@openshift-console)/)/'"
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'"
```

Comment:
No need to transpile `@openshift-console` package sources anymore since we're already consuming the distributable plugin SDK packages.